packages/address-book-controller/src/AddressBookController-method-action-types.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * This file is auto generated by `scripts/generate-method-action-types.ts`.
+ * Do not edit manually.
+ */
+import type { AddressBookController } from './AddressBookController';
+
+/**
+ * Returns all address book entries as an array.
+ *
+ * @returns Array of all address book entries.
+ */
+export type AddressBookControllerListAction = {
+  type: `AddressBookController:list`;
+  handler: AddressBookController['list'];
+};
```

Comment:
What are your thoughts on updating the script to add empty line breaks between action definitions?